wpeverest/user-registration
# Fix - Token mismatch issue

### All Submissions:






### Changes proposed in this Pull Request:

This PR fixes, Token Mismatch Error was shown while email confirmation link clicked. This issue occurs only for some users only. I have provided this code to a user issue that was fixed on the user's site.

### How to test the changes in this Pull Request:

1. Register User with email confirmation login option.
2. Now try to click on the confirmation link received in the email.
3. It should verify successfully.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Token mismatch issue
